docs/source/examples/predict_next_purchase.ipynb

```diff
@@ -188,7 +187,7 @@
     "\n",
     "### Representing the Data\n",
     "\n",
-    "We will represent the online grocery orders using an entity set. This way, we can generate features based on the relational structure of the dataset. We currently have a single table of orders where one customer can many orders. This one-to-many relationship can be represented by normalizing an entity for customers. The same can be done for departments, aisles, and products. Let's structure the entity set."
+    "Let's start by representing the data with an entity set. This way, we can generate features based on the relational structure of the dataset. We currently have a single table of orders where one customer can many orders. This one-to-many relationship can be represented by normalizing a customer entity. The same can be done for other one-to-many relationships like aisle-to-products. We want to make predictions based on the customer, so we will use this customer entity as the target entity for generating features."
```

Comment:
`We currently have a single table of orders where one customer can many orders.`

This sentence is missing a word - maybe this instead `...where one customer can have many orders.`